spec/defines/dropin_file_spec.rb

```diff
@@ -36,6 +36,21 @@
             }.to raise_error(/expects a match for Systemd::Dropin/)
           }
         end
+
+        context 'with explicit filename' do
+            let (:title) {'test'}
```

Comment:
Indenting is a bit odd here

Reply:
you are absolutely right! 
I'll fix that and create a test for the same drop-in file for two services.

Reply:
I've fixed the indenting and changed unit-test to the explicit test you suggested.